ServerlessBenchmark/ServerlessPlatformControllers/AWS/AWS_Controller.cs

```diff
@@ -16,6 +16,13 @@ namespace ServerlessBenchmark.ServerlessPlatformControllers.AWS
 {
     public class AwsController: ICloudPlatformController
     {
+        private ILogger _logger;
+
+        public AwsController(ILogger logger)
+        {
+            this._logger = logger;
+        }
```

Comment:
Also do this for AzureController class. 
Maybe you can abstract the constructor away in a base class and initialize the logger there.